README.md

```diff
@@ -86,7 +86,7 @@ Vizro-AI is a separate package and extends Vizro to enable the use of natural la
 <img src="./vizro-ai/docs/assets/readme/readme_animation.gif" alt="Gif to show vizro-ai", width="525" height="296">
 </p>
 
-See the [Vizro-AI documentation](https://vizro.readthedocs.io/projects/vizro-ai/en/latest/) for more details.
+See the [Vizro-AI documentation](https://vizro-ai.readthedocs.io/) for more details.
```

Comment:
I would prefer that we used  **https://vizro.readthedocs.io/projects/vizro-ai/** rather than the **vizro-ai.rtd.io** link, because it's the correct subproject notation (as in, Vizro-AI is a subproject of Vizro). I know they're set up separately as projects, but by keeping them connected in the docs under one domain (vizro.readthedocs.io) means that we can later use our own domain easily (eg vizro.org) and map things smoothly to it. We did this with Kedro and Kedro-Viz docs.

WDYT?

Reply:
<img width="946" alt="Screenshot 2024-08-08 at 13 01 57" src="https://github.com/user-attachments/assets/8c6cae0a-b24e-44df-a622-2c4d726f6806">

Reply:
Ok agreed, let me change what I just changed to that then

Reply:
Done